README.md

```diff
@@ -208,25 +208,36 @@ The VC++ linker supports `/SOURCELINK` [switch](https://docs.microsoft.com/en-us
 
 If you distribute the library via a package published to [NuGet.org](http://nuget.org), it is recommended to build a [symbol package](https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg) and publish it to [NuGet.org](http://nuget.org) as well. This will make the symbols available on [NuGet.org symbol server](https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg#nugetorg-symbol-server), where the debugger can download it from when needed.
 
-Alternatively, Portable PDBs can be included in the main NuGet package by setting the following property in your project:
-
-```xml
-  <PropertyGroup>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-  </PropertyGroup>  
-```
-
-Keep in mind that including PDBs in the .nupkg increases the size of the package and thus restore time for projects that consume your package, regardless of whether the user needs to debug through the source code of your library or not.
-
 .snupkg symbol packages have following limitations:
 
 - They do not support Windows PDBs (generated by VC++, or for managed projects that set build property `DebugType` to `full`)
 - They require the library to be built by newer C#/VB compiler (Visual Studio 2017 Update 9).
 - The consumer of the package also needs Visual Studio 2017 Update 9 debugger.
 - Not supported by [Azure DevOps Artifacts](https://azure.microsoft.com/en-us/services/devops/artifacts) service.
 
-Consider including PDBs in the main package if it is not possible to use .snupkg for the above reasons. 
-For managed projects, consider switching to Portable PDBs by setting `DebugType` property to `portable`. This is the default for .NET SDK projects, but not classic .NET projects.
+If a .snupkg does not work for your scenario, consider including debug information in the main package.
+
+> [!IMPORTANT]
+> Keep in mind that including debug information in the .nupkg increases the size of the package and thus restore time for projects that consume your package, regardless of whether the user needs to debug the source code of your library or not.
+
+The simplest way to include debug information is to embed Portable PDBs directly in the assembly by setting the following property in your project:
+
+```xml
+<PropertyGroup>
+  <DebugType>embedded</DebugType>
+</PropertyGroup>
+```
+
+Alternatively, you can include PDB files in the main NuGet package by setting the following property in your project:
+
+```xml
```

Comment:
I would suggest including portable PDBs in the nupkg first and embedding PDBs into the assembly as an alternative. The latter increases size of the assembly, which may potentially be an issue in some cases.